MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -268,17 +268,14 @@ private async Task SavePeopleMetadataAsync(List<PersonInfo> people, LibraryOptio
 
         private async Task AddPersonImageAsync(Person personEntity, LibraryOptions libraryOptions, string imageUrl, CancellationToken cancellationToken)
         {
-            if (libraryOptions.DownloadImagesInAdvance)
+            try
             {
-                try
-                {
-                    await ProviderManager.SaveImage(personEntity, imageUrl, ImageType.Primary, null, cancellationToken).ConfigureAwait(false);
-                    return;
-                }
-                catch (Exception ex)
-                {
-                    Logger.LogError(ex, "Error in AddPersonImage");
-                }
+                await ProviderManager.SaveImage(personEntity, imageUrl, ImageType.Primary, null, cancellationToken).ConfigureAwait(false);
```

Comment:
We should remove this because of
1. Entity will be saved: https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Providers/Manager/MetadataService.cs#L255-L263
2. UpdateToRepositoryAsync will call https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Library/LibraryManager.cs#L1867
3. Images will be downloaded :)